x-pack/plugins/security_solution/public/management/pages/trusted_apps/store/middleware.ts

```diff
@@ -139,9 +145,16 @@ const submitCreationIfNeeded = async (
   store: ImmutableMiddlewareAPI<TrustedAppsListPageState, AppAction>,
   trustedAppsService: TrustedAppsService
 ) => {
-  const submissionResourceState = getCreationSubmissionResourceState(store.getState());
-  const isValid = isCreationDialogFormValid(store.getState());
-  const entry = getCreationDialogFormEntry(store.getState());
+  const currentState = store.getState();
+  const submissionResourceState = getCreationSubmissionResourceState(currentState);
+  const isValid = isCreationDialogFormValid(currentState);
+  const entry = getCreationDialogFormEntry(currentState);
+  const editMode = isEdit(currentState);
+
+  // FIXME: Implement PUT API for updating Trusted App - https://github.com/elastic/security-team/issues/682#issuecomment-768581449
```

Comment:
are you all ok with this?
Its a feature branch ++ I did not want this PR to get too large and would like to iterate on it

Reply:
I don't think that URL can be in here

Reply:
yeah. Fixed 😄 thanks